docs/How  to use the epos plugin.md

```diff
@@ -0,0 +1,136 @@
+# How to use the Epos plugin of the FAIR evaluator 
+To use the fair evalater you need access to a terminal 
+
+## 1.Follow the steps indicated in the  documentation
+https://github.com/EOSC-synergy/FAIR_eva/blob/main/docs/index.md which are executting the following commands
+
+git clone https://github.com/EOSC-synergy/FAIR_eva.git
+cd ./FAIR_eva
+pip3 install -r requirements.txt
+cp config.ini.template config.ini
```

Comment:
In the case of EPOS it is `plugins/epos/config.ini`

Reply:
I understand that the config.ini in pluggins doesnt have a template

Reply:
There is no template in any of the plugins

Reply:
Is this (main) config.ini required for running epos plugin? If not, I would remove the line

Reply:
Yes it is i have tested it righ now